app/src/components/BaseFilterNav.vue

```diff
@@ -0,0 +1,64 @@
+<template>
```

Comment:
Also getting some errors in the console (likely unrelated to this file, just using it to start a thread)

![image](https://user-images.githubusercontent.com/17163988/131416487-e3e4457a-f299-4609-b194-3ea17e5f3f57.png)

Reply:
I think I've got this resolved, can you let me know if this is fixed for you?

Reply:
Yes I no longer see it!